src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/modifier/interfacememberimpliedmodifier/InputInterfaceMemberImpliedModifierSealedClasses.java

```diff
@@ -0,0 +1,55 @@
+/*
+InterfaceMemberImpliedModifier
+
```

Comment:
I just noticed this now when reviewing another PR. How come this top portion doesn't mention the default properties?

I guess our tool doesn't ensure all properties are listed. We are missing double validation here.

Reply:
Yeah done, I add the default properties

> I guess our tool doesn't ensure all properties are listed.  

yes

Reply:
Better to reference them, even it is not enforced